pkgs/tools/admin/azure-cli/default.nix

```diff
@@ -254,7 +254,7 @@ py.pkgs.toPythonApplication (py.pkgs.buildAzureCliPackage {
   '' + lib.optionalString (withExtensions != [ ]) ''
     --set AZURE_EXTENSION_DIR ${extensionDir} \
   '' + ''
-    --set PYTHONPATH $PYTHONPATH
+    --set PYTHONPATH "${python3.pkgs.makePythonPath propagatedBuildInputs}:$out/${python3.sitePackages}"
```

Comment:
We should probably make this a fixup of the original commit

Reply:
I didn't introduce or changed that line in a previous commit, it was only moved (that's why GitHub is showing it as added), so I thought a separate commit for a fix that is otherwise unrelated to this PR would make sense.